libs/langchain/langchain/vectorstores/pgvector.py

```diff
@@ -108,6 +109,7 @@ def __init__(
         self,
         connection_string: str,
         embedding_function: Embeddings,
+        engine_args: Optional[dict[str, Any]] = None,
```

Comment:
@aubinmazet this is a breaking change since users may have been invoking this positionally, could we add at end of list and use a `*` preceding it?

Reply:
Updated the argument to a  **kwargs syntax